src/mountebank.js

```diff
@@ -148,6 +148,54 @@ function create (options) {
         });
     });
 
+    function portStored (saveImpostersFile, saveImpostersFileFlag) {
+        var fs = require('fs');
+        var portStore = [];
+        var ImposterDir = './Repository_Template';
```

Comment:
*  We hard-coded just the folder name to avoid confusion of users to be saved but still file names are flexible .
*  **What we can do to avoid hard coding** is to get directory name from the user itself like `mb --savefile  mb123.json --dir repo123`.
*  Another reason why we need proper directory structure is we are reading and parsing json data from the files to retain the older data
1. For example if we save file using command `mb --savefile mb123.json` and started to save data and 3 post requests data was stored. Now next time if you restart the mb the older data should remain and the new data should be added on top of that and avoid duplication entry.


Let us know what you think and we can make changes accordingly.

Reply:
Ah, I'd expect it to just be mb --savefile repo123/mb123.json. In other words the savefile is just a file path.